scripts/codecov.sh

```diff
@@ -35,22 +35,14 @@ GO_TOP=$(cd "$(dirname "$0")"/../../../..; pwd)
 
 export GOPATH=${GOPATH:-$GO_TOP}
 
-if [ "${1:-}" != "" ]; then
+if [[ "${1:-}" != "" ]]; then
```

Comment:
should this be fixed also in istio/istio? 
that's what happens when you touch something ;-)

Reply:
thanks for reminder, I would fix related problems in istio/istio